beacon_node/http_api/src/lib.rs

```diff
@@ -1349,8 +1351,8 @@ pub fn serve<T: BeaconChainTypes>(
              chain: Arc<BeaconChain<T>>,
              network_tx: UnboundedSender<NetworkMessage<T::EthSpec>>,
              log: Logger| {
-                task_spawner.spawn_async(Priority::P1, async move {
-                    match publish_blocks::publish_block(
+                task_spawner.spawn_async_with_rejection(Priority::P0, async move {
```

Comment:
This is the priority bump

Reply:
Nice catch, not sure how that slipped through 🙏